packages/sentry/src/sentry-user-logout-tracker/sentry-user-logout-tracker.tsx

```diff
@@ -1,7 +1,7 @@
 import type { ApplicationWindow } from '@commercetools-frontend/constants';
 
 import { Component } from 'react';
-import * as Sentry from '@sentry/browser';
+import * as Sentry from '@sentry/react';
```

Comment:
Just a tiny wrapper around `@sentry/browser` with some more specific [React features](https://docs.sentry.io/platforms/javascript/guides/react/features/) (in case we want to use them).